requirements/edx/base.txt

```diff
@@ -47,7 +47,7 @@ edx-drf-extensions==1.2.2
 edx-lint==0.4.3
 edx-django-oauth2-provider==1.1.4
 edx-django-sites-extensions==2.1.1
-edx-enterprise==0.9.0
+git+https://github.com/open-craft/edx-enterprise.git@8849481663809f0190545cbd6e69c37be1904872#egg=edx-enterprise==0.19.0
```

Comment:
We'll need to update this reference to point to the official version once available.